pytorch_toolkit/README.md

```diff
@@ -39,7 +39,7 @@ them for prediction.
 * [Segmentation of Thoracic Organs](segthor)
 * [Super Resolution](super_resolution)
 * [Formula recognition](formula_recognition)
-* [Text Spotting](text_spotting)
+* [Text Spotting](text_spotting_2)
```

Comment:
Is needed `segmentoly` with new `text_spotting` models based on `mmdetection`?

Reply:
I am going to remove old 'instance_segmentation' (segmentoly) and 'text_spotting' in a next PR. This PR is already quite big.

Reply:
OK.